src/common/hotbar-store.ts

```diff
@@ -89,18 +68,16 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
     return this.hotbarIndex(this.activeHotbarId);
   }
 
-  static getInitialItems() {
-    return [...Array.from(Array(defaultHotbarCells).fill(null))];
-  }
-
   @action
   protected fromStore(data: Partial<HotbarStoreModel> = {}) {
     if (!data.hotbars || !data.hotbars.length) {
-      this.hotbars = [{
-        id: uuid.v4(),
-        name: "Default",
-        items: this.defaultHotbarInitialItems,
-      }];
+      const hotbar = getEmptyHotbar("default");
```

Comment:
Was "Default", now is "default" ? I guess we've always set or required the name so "Default" never actually appears anywhere?

Reply:
Good catch, I think I should leave it as `Default` so as to not change.